src/app/pages/skill-page/components/skill/skill.component.html

```diff
@@ -0,0 +1,9 @@
+<img
+  class="skill-icon"
+  src="/assets/skill_icons/skill_{{ this.skillID }}{{ this.maxed ? '_maxed' : undefined }}.gif" />
```

Comment:
You could replace `undefined` with `''`.
One might think that it will print out undefined instead of an empty string

Reply:
fixed